pygmt/helpers/decorators.py

```diff
@@ -17,15 +17,18 @@
         region : str or list
             *Required if this is the first plot command*.
             *xmin/xmax/ymin/ymax*\ [**+r**][**+u**\ *unit*].
-            Specify the region of interest.""",
+            Specify the region of interest.
+            Select map :doc:`region </tutorials/regions>`.""",
```

Comment:
You can combine these two sentences.
```suggestion
            Specify the :doc:`region </tutorials/regions>` of interest.""",
```